scripts/cue-test/cue-test.go

```diff
@@ -65,3 +62,67 @@ func main() {
 		return sch.ValidateVariable(plugin, name)
 	})
 }
+
+func validateAllDashboards(sch schemas.Schemas) {
+	logrus.Info("validate all dashboard in dev/data")
+	data, err := os.ReadFile(path.Join("dev", "data", "dashboard.json"))
```

Comment:
Looking at this again it, it makes sense, I see we're validating the initial data that gets populated, I think I'd be able to iterate on panel properties in local_db still?